third_party/opa/data-and-policies/user-created-policy-3/sample_policies.rego

```diff
@@ -0,0 +1,82 @@
+package dataapi.authz
+
+verdict[output] {
+	count(rule) == 0
+	output = {"action": {"name":"Deny", "Deny": {"Information": "Access is denied"}}, "policy": "Deny by default"}
```

Comment:
No need for Information field (also forbidden by current example taxonomy)
```suggestion
	output = {"action": {"name":"Deny", "Deny": {}}, "policy": "Deny by default"}
```

Reply:
Removed it.